docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
 
     prometheus:
         image: functions/prometheus:latest  # autobuild from Dockerfile in repo.
-        command: "-config.file=/etc/prometheus/prometheus.yml -storage.local.path=/prometheus -storage.local.memory-chunks=10000 --alertmanager.url=http://alertmanager:9093"
+        command: "--config.file=/etc/prometheus/prometheus.yml"
```

Comment:
Where is the `--alertmanager.url=http://alertmanager:9093` flag? is it deprecated?

Reply:
The alertmanager target moved to prometheus.yaml:

```
    alerting:
      alertmanagers:
      - static_configs:
        - targets:
          - alertmanager:9093
```